sqlmesh/cli/main.py

```diff
@@ -1007,3 +1007,22 @@ def dlt_refresh(
 def environments(obj: Context) -> None:
     """Prints the list of SQLMesh environments with its expiry datetime."""
     obj.print_environment_names()
+
+
+@cli.command("lint")
+@click.option(
+    "--model",
+    "--models",
+    multiple=True,
+    help="A model to lint. Multiple models can be linted.",
+)
+@click.pass_obj
+@error_handler
+@cli_analytics
+def lint(
+    obj: Context,
+    models: t.Iterator[str],
+) -> None:
+    """Run the linter for the target model(s)."""
+    model_list = list(obj.get_model(model) for model in models) if models else obj.models.values()
+    obj.lint_models(*model_list)
```

Comment:
Why are we constructing the model list here instead of `lint_models`? If you do this you'll have to repeat the logic in the magics, etc.